core/tests/protractor/profileMenuFlow.js

```diff
@@ -85,6 +87,38 @@ describe('Profile menu flow', function() {
         expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/learner_dashboard');
       });
 
+    it('should not show the topics and skills dashboard link in the profile ' +
+      'dropdown menu when user is not admin', function() {
+      element.all(
+        by.css(
+          '.protractor-test-topics-and-skills-dashboard-link')
+      ).then(function(links) {
+        expect(links.length).toEqual(0);
+      });
+    });
+
+    it('should visit the topics and skills dashboard from the profile ' +
+      'dropdown menu when user is admin', function() {
+      users.logout();
+
+      users.login('desktopAndMobileAdm@profileMenuFlow.com');
+      learnerDashboardPage.get();
+      var profileDropdown = element(by.css(
+        '.protractor-test-profile-dropdown'));
+      waitFor.elementToBeClickable(
+        profileDropdown, 'Could not click profile dropdown');
+      profileDropdown.click();
+
+      var topicsAndSkillsDashboardLink = element(by.css(
+        '.protractor-test-topics-and-skills-dashboard-link'));
+      waitFor.elementToBeClickable(
+        topicsAndSkillsDashboardLink,
+        'Could not click on the topics and skills dashboard link');
+      topicsAndSkillsDashboardLink.click();
+      waitFor.pageToFullyLoad();
+      expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/topics_and_skills_dashboard');
```

Comment:
Just a confirmation... Is localhost fine in this URL?

Reply:
Yeah, it's the same format followed by the tests for learner and creator dashboard.